documentation/modules/ROOT/pages/connectors/vitess.adoc

```diff
@@ -1006,6 +1054,78 @@ a|_n/a_
 
 |===
 
+[id="vitess-temporal-types"]
+=== Temporal types
+
+Excluding the `TIMESTAMP` data type, Vitess temporal types depend on the value of the `time.precision.mode` connector configuration property.
+
+.Temporal values without time zones
+The `DATETIME` type represents a local date and time such as "2018-01-13 09:48:27". As you can see, there is no time zone information. Such columns are converted into epoch milliseconds or microseconds based on the column’s precision by using UTC. The `TIMESTAMP` type represents a timestamp without time zone information. It is converted by MySQL from the server (or session’s) current time zone into UTC when writing and from UTC into the server (or session's) current time zone when reading back the value. For example:
+
+* `DATETIME` with a value of `2018-06-20 06:37:03` becomes `1529476623000`.
+* `TIMESTAMP` with a value of `2018-06-20 06:37:03` becomes `2018-06-20T13:37:03Z`.
+
+Such columns are converted into an equivalent `io.debezium.time.ZonedTimestamp` in UTC based on the server (or session’s) current time zone. The time zone will be queried from the server by default. If this fails, it must be specified explicitly by the database `connectionTimeZone` MySQL configuration option. For example, if the database’s time zone (either globally or configured for the connector by means of the `connectionTimeZone` option) is "America/Los_Angeles", the TIMESTAMP value "2018-06-20 06:37:03" is represented by a `ZonedTimestamp` with the value "2018-06-20T13:37:03Z".
+
+The time zone of the JVM running Kafka Connect and Debezium does not affect these conversions.
+
+More details about properties related to temporal values are in the documentation for xref:vitess-connector-properties[Connector configuration properties].
+
+time.precision.mode=adaptive_time_microseconds(default)::
+The Vitess connector determines the literal type and semantic type based on the column's data type definition so that events represent exactly the values in the database. All time fields are in microseconds. Only positive `TIME` field values in the range of `00:00:00.000000` to `23:59:59.999999` can be captured correctly.
```

Comment:
```suggestion
The Vitess connector determines the literal type and semantic type based on the column's data type definition so that events represent exactly the values in the database. 
All time fields are in microseconds. 
Only positive `TIME` field values in the range of `00:00:00.000000` to `23:59:59.999999` can be captured correctly.
```